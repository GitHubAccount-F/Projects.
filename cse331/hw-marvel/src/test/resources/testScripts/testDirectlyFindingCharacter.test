#Starts from one Marvel Character, and goes directly to
#the wanted Marvel character(no alternative paths to explore, just the
#path that leads to that character).

#Creates the graph
LoadGraph graph1 directPathInSameComic.csv

#Finds the path between characters
FindPath graph1 SpiderMan MilesMorales
FindPath graph1 Venom Carnage