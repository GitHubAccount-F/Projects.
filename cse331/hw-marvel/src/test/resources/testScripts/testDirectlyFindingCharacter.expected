#Starts from one Marvel Character, and goes directly to
#the wanted Marvel character(no alternative paths to explore, just the
#path that leads to that character).

#Creates the graph
loaded graph graph1

#Finds the path between characters
path from SpiderMan to MilesMorales:
SpiderMan to MilesMorales via Spider-Verse
path from Venom to Carnage:
Venom to Carnage via Spider-Verse